components/switch/src/switch.js

```diff
@@ -0,0 +1,165 @@
+/** @jsx jsx */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme, paint } from '@westpac/core';
+
+import { SwitchText, SwitchToggle } from './styled';
+
+// ==============================
+// Utils
+// ==============================
+
+// ==============================
+// Component
+// ==============================
+
+export const Switch = ({
+	name,
+	id,
+	toggleText,
+	size,
+	block,
+	flip,
+	srOnlyText,
+	defaultChecked,
```

Comment:
should be checked since defaultChecked will only set the initial value and leave it up to the html element to manage its internal state (uncontrolled component) which we dont want.